src/internal/Subscriber.ts

```diff
@@ -184,22 +174,6 @@ function handleStoppedNotification(notification: ObservableNotification<any>, su
   onStoppedNotification && timeoutProvider.setTimeout(() => onStoppedNotification(notification, subscriber));
 }
 
-/**
- * The observer used as a stub for subscriptions where the user did not
- * pass any arguments to `subscribe`. Comes with the default error handling
- * behavior.
- */
-export const EMPTY_OBSERVER: Readonly<Observer<any>> & { closed: true } = {
-  closed: true,
-  next: noop,
-  error: defaultErrorHandler,
-  complete: noop,
-};
-
-function isObserver<T>(value: any): value is Observer<T> {
-  return value && isFunction(value.next) && isFunction(value.error) && isFunction(value.complete);
-}
-
-export function isSubscriber<T>(value: any): value is Subscriber<T> {
```

Comment:
This isn't useful. The only way we can guarantee safety is applied during subscription is to check that it's OUR instance of `Subscriber`. Removed in favor of a simple `instanceof` check.

Reply:
Are we worried about what happens if someone has two versions of Rx in the same app? i.e. some random package returns an `rxjs@v6` subscription that is added to an `rxjs@v8` subscriber. Or is that handled by always wrapping the unknown observer via `createSafeObserver()`?

Reply:
I think removing `isSubscriber()` makes sense. `createSafeObserver()` should yield normalized observer behavior(s).

Reply:
@trxcllnt it will wrap it in the safe observer. There's a possibility that in a mixed environment mixing one RxJS's copy of Observable, and another's copy of some operator might result in subscriptions not being chained properly, but I haven't confirmed that. In any case, that's a corner case I wouldn't want to punish everyone else for in terms of size. I'll research that to see if it's even possible. If there's a small fix that doesn't cost everyone else, then we'll fix it. If not, we'll have to document workarounds, etc.